README.md

```diff
@@ -80,6 +80,35 @@ make mysoc_myboard_gp_mmc_defconfig
 make
 ```
 
+### Building an SD card image
+
+The K3 ROM when booting from an SD card in filesystem boot mode can be very
+particular about the format it expects. To ease generation of a working
+SD card this tool can provide an simple SD card image.
+
+To build an SD card image your host system will need mtools installed:
+
+```
+sudo apt install mtools
```

Comment:
parted and dosfstools ?

Reply:
Installed by default in Debian/Ubuntu, but agree will add these explicitly for completeness.